R/grid_search.R

```diff
@@ -122,18 +151,18 @@ scan_beta_date <- function(
 }
 
 ##' @export
-plot.sircovid_scan <- function(x, ..., what = "likelihood") {
+plot.sircovid_scan <- function(x, ..., what = "likelihood", date = as.character(Sys.Date())) {
   if (what == "likelihood") {
     graphics::image(x=x$x, y=x$y, z=x$mat_log_ll,
-                    xlab="beta", ylab="Start date", main = "Log-likelihood")
+                    xlab="beta", ylab="Start date", main = paste0("Log-likelihood (run ", date ,")"))
   } else if (what == "probability") {
     graphics::image(x=x$x, y=x$y, z=x$renorm_mat_LL,
-                    xlab="beta", ylab="Start date", main = "Probability")
+                    xlab="beta", ylab="Start date", main = paste0("Probability (run ", date ,")"))
   }
 }
 
 ##' @export
-plot.sample_grid_search <- function(x, ..., what = "ICU") {
+plot.sample_grid_search <- function(x, ..., what = "ICU", date = as.character(Sys.Date())) {
```

Comment:
I think that we should be explicit about date throughout or we will get into trouble where the date of the *data* is not today's date.  That will happen fairly frequently (impacts at least the other plotting function too).  I spent some time removing this from a different piece of work unrelated to this workstream...

Reply:
Thanks, have changed it so that it can be set manually if wanted